src/components/Header/Header.tsx

```diff
@@ -32,18 +34,17 @@ export const Header = () => {
           />
           <Styled.MobileSearchInput
             value={searchValue}
-            onChange={(evt) => setSearchValue(evt.target.value)}
+            onChange={(evt: React.ChangeEvent<HTMLInputElement>) =>
+              setSearchValue(evt.target.value)
+            }
             onSearch={() => history.push(`/search?q=${searchValue}`)}
             width="80%"
           />
         </>
       ) : (
         <>
           <FlexDiv align="center">
-            <Styled.Hamburger
-              active={showMenu}
-              onClick={() => setShowMenu(!showMenu)}
-            >
+            <Styled.Hamburger active={showMenu} onClick={() => setShowMenu(!showMenu)}>
```

Comment:
When I need to update a **boolean** value in `useState` hook I prefer the following way just to make sure we always have the last state before change it:
```
setShowMenu((showMenu) => !showMenu)
```
This can be a good example about it: https://codesandbox.io/s/stack-overflow-demo-nmjiy?file=/src/App.js